app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -231,9 +231,12 @@ class TableVisualization extends Visualization {
             const height = this.dom.getAttributeNS(null, 'height')
             const tblViewStyle = `width: ${width}px; height: ${height}px; overflow: scroll;`
             this.tabElem.setAttributeNS(null, 'style', tblViewStyle)
+            // Ensure that this.tabElem and this.dom have been updated before calling
+            // sizeColumnsToFit() to get the latest element styles.
```

Comment:
Could you please describe it better. If I did not know what this is about, after reading this comment in the code I would not understand it at all. Include in the description info why this is here, how it affects visualization and why it affects it so, please :)

Reply:
This is a problem caused by browser reflow. I searched for a long time but couldn't find the demo I had seen before, and I forgot how to reproduce it. However, I will try my best to explain it in the comments.

Reply:
This even more shows that this doc comment should be more clear, as a potential reader in the future should be able to not only understand it, but should know how to check it / reproduce it. If we have problem with that already, we need to improve it definitely :)